rust/core-lib/src/rpc.rs

```diff
@@ -281,6 +284,15 @@ impl<'a> EditCommand<'a> {
             "redo" => Ok(Redo),
             "cut" => Ok(Cut),
             "copy" => Ok(Copy),
+
+            "find" => params.as_object().map(|dict| {
+                let chars = dict_get_string(dict, "chars");
+                let case_sensitive = dict_get_bool(dict, "case_sensitive").unwrap_or(false);
```

Comment:
You can probably use the newer serde-json features to make this more concise, something like `dict["case_sensitive"].as_bool().unwrap_or(false)`